app/Services/Customer/OrderService.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Services\Customer;
+
+use App\Repositories\Eloquent\OrderDetailRepository;
+use App\Repositories\OrderRepositoryInterface;
+
+
+class OrderService
+{
+    private OrderRepositoryInterface $orderRepository;
+    private OrderDetailRepository $orderDetailRepository;
+
+    public function __construct(OrderRepositoryInterface $orderRepository,OrderDetailRepository $orderDetailRepository)
+    {
+        $this->orderRepository = $orderRepository;
+        $this->orderDetailRepository = $orderDetailRepository;
+    }
+
+    //Create order
+    public function createOrder($request)
+    {
+        return $this->orderRepository->create($request);
+    }
+
+    //Create order details
+    public function createOrderDetail($request)
+    {
+        return $this->orderDetailRepository->create($request);
+    }
+
+    //Find order by id
+    public function findById($id)
+    {
+        return $this->orderRepository->findById($id);
+    }
+
+    //Update order status
+    public function updateStatus($id, $request)
+    {
+        try {
+            $array = ['status' => $request->status ];
+            $this->orderRepository->update($id,$array);
+            return true;
+        }catch (\Exception $ex)
+        {
+            return false;
+        }
+    }
+
+    //Delete order
+    public function deleteOrder($id)
+    {
+        try {
+            $order = $this->findById($id);
```

Comment:
when execute multiple tables, please use DB::transaction to maintain DB when error